lib/src/cache_manager.dart

```diff
@@ -135,8 +135,13 @@ class FirebaseImageCacheManager {
   }
 
   Future<Uint8List> remoteFileBytes(
-      FirebaseImageObject object, int maxSizeBytes) {
-    return object.reference.getData(maxSizeBytes);
+      FirebaseImageObject object, int maxSizeBytes) async {
```

Comment:
Now, `maxSizeBytes` has no purpose and should be removed.

But really, rather than changing `remoteFileBytes`, consider adding an alternative method without the restriction.

Alternatively, consider keeping the `object.reference.getData(maxSizeBytes)` call _when_ `maxSizeBytes` is specified, and doing the new logic when it isn't.